packages/grid/_modules_/grid/components/toolbar/GridToolbarExport.tsx

```diff
@@ -0,0 +1,76 @@
+import * as React from 'react';
+import MenuList from '@material-ui/core/MenuList';
+import Button from '@material-ui/core/Button';
+import MenuItem from '@material-ui/core/MenuItem';
+import { ApiContext } from '../api-context';
+import { useGridSelector } from '../../hooks/features/core/useGridSelector';
+import { optionsSelector } from '../../hooks/utils/optionsSelector';
+import { GridMenu } from '../menu/GridMenu';
+import { ExportOption } from '../../models';
+
+export function GridToolbarExport() {
+  const apiRef = React.useContext(ApiContext);
+  const options = useGridSelector(apiRef, optionsSelector);
+  const [anchorEl, setAnchorEl] = React.useState(null);
+  const ExportIcon = apiRef!.current.components!.ExportIcon!;
+
+  const ExportOptions: Array<ExportOption> = [
+    {
+      label: apiRef!.current.getLocaleText('toolbarExportCSV'),
+      format: 'csv',
+    },
+  ];
+
+  const handleExportSelectorOpen = (event) => setAnchorEl(event.currentTarget);
+  const handleExportSelectorClose = () => setAnchorEl(null);
+  const handleExport = (format) => {
+    if (format === 'csv') {
+      apiRef!.current.exportDataAsCsv();
+    }
+
+    setAnchorEl(null);
+  };
+
+  const handleListKeyDown = (event: React.KeyboardEvent) => {
+    if (event.key === 'Tab' || event.key === 'Escape') {
+      event.preventDefault();
+      handleExportSelectorClose();
+    }
+  };
+
+  // Disable the button if the corresponding is disabled
+  if (options.disableCsvExport) {
+    return null;
+  }
+
+  const renderExportOptions: Array<React.ReactElement> = ExportOptions.map((option, index) => (
```

Comment:
I think it adds a bit of overhead. We just manage 1 format. We are not sure if we will manage more. No reason to write extra code now

Reply:
> We are not sure if we will manage more

Are you saying you envision a world where we won't support Excel exports? It seems to be an important feature for the premium plan.

Reply:
I did it like this because in the docs there are multiple sections for different formats so it seemed like a good idea to prep it for future extension.

Reply:
I'm not saying that we won't support xlsx, I'm just saying that until we do, we don't need that code. So we can just refactor to support more format later. Also, atm, it could just go directly as children of the menuList as it's a static length, 1, max 3 items.
Lastly the onClick handler is recreated at each render

Reply:
> the onClick handler is recreated at each render

Whish looks great. I thin that any usage useCallback should be justified with a benchmark if it's performance-related or else, e.g. not dropping events with an event listener. This isn't one of these cases, unless proven otherwise.

Reply:
If there are no solid arguments against it I would keep it as it is because it also follows the same mental model as the `DensitySelector` so in that regard it is consistent.